docs/hacking/administration.rst

```diff
@@ -0,0 +1,18 @@
+Administration
+--------------
+
+To access the admin interface, a user must be logged into hypothes.is and have admin permissions.
+To grant admin permissions to a user, run the following command in the virtual environment:
+
+	.. code-block:: bash
+	
+	hypothesis admin <config_uri> <username>
+
```

Comment:
These are still not rendering correctly, I think you indented them too far. The `.. code-block:: bash` line should be o lines indented, the other one should be 3 lines indented